formula_sae_esf.tex

```diff
@@ -1585,7 +1582,7 @@ \section{Additional LV-parts Interfering with the Tractive System}
                 \label{dashschem}
             \end{figure}
 
-            The dashboard node is wired to have the steering angle potentiometer and E-stops as input and the control panel interface (the LED) as an output. It listens to all CAN messages around the car. The interfacing LED will notify the driver of anything deemed important, such as the AMS detecting the temperature of the accumulator cells was getting too high. Therefore, the dashboard's interfacing light will mostly be telling the driver why the shutdown system shut down through a series of flashes. Only if one of the E-stops are pressed will the CAN system shut down and therefore the interfacing light will not tell the driver what has happened to the shutdown circuit.
+            The dashboard node is wired to have the \hlr{E-stops as input} and the control panel interface (the LED) as an output. It listens to all CAN messages around the car. The interfacing LED will notify the driver of anything deemed important, such as the AMS detecting the temperature of the accumulator cells was getting too high. Therefore, the dashboard's interfacing light will mostly be telling the driver why the shutdown system shut down through a series of flashes. Only if one of the E-stops are pressed will the CAN system shut down and therefore the interfacing light will not tell the driver what has happened to the shutdown circuit.
```

Comment:
What is this interfacing light? We have dedicated lights for the IMD and AMS errors, but we have and LCD display to communicate with the driver.